04.class/lib/view.js

```diff
@@ -0,0 +1,29 @@
+import enquirer from "enquirer";
+
+export class View {
+  #MODE_TABLE = { r: "see", d: "delete" };
+
+  printMemos(memos) {
+    memos.forEach((memo) => {
+      console.log(memo.title);
+    });
+  }
+
+  displayMemoSelectionPrompt(memos, mode) {
+    const titles = memos.map((memo) => memo.title);
+    const prompt = new enquirer.Select({
+      name: "memo",
+      message: `Choose a memo you want to ${this.#MODE_TABLE[mode]}:`,
```

Comment:
```suggestion
      message: `Choose a memo you want to ${act}:`,
```

引数でもらってきた文字列をそのまま表示する、でもいい気がします